src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -509,8 +510,7 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
   private initQueryEvents() {
     this.bind.onRootElement(QueryEvents.duringQuery, () => this.ensureDom());
     this.bind.onRootElement(QueryEvents.doneBuildingQuery, (data: IDoneBuildingQueryEventArgs) => this.handleDoneBuildingQuery(data));
-    this.bind.onRootElement(QueryEvents.querySuccess, (data: IQuerySuccessEventArgs) => this.handleQuerySuccess(data));
-    this.bind.onRootElement(QueryEvents.deferredQuerySuccess, () => this.handleDeferredQuerySuccess());
```

Comment:
This was put on because of the padding feature but now it's not there anymore hehe and everything updates nicely at the same time :-)